prover/crates/bin/prover_job_monitor/src/queue_reporter/prover_queue_reporter.rs

```diff
@@ -0,0 +1,83 @@
+use async_trait::async_trait;
+use zksync_config::configs::fri_prover_group::FriProverGroupConfig;
+use zksync_prover_dal::{Connection, Prover, ProverDal};
+use zksync_types::{basic_fri_types::CircuitIdRoundTuple, prover_dal::JobCountStatistics};
+
+use crate::{
+    metrics::{JobStatus, PROVER_JOB_MONITOR_METRICS},
+    task_wiring::Task,
+};
+
+/// `ProverQueueReporter` is a task that reports prover jobs status.
+/// Note: these values will be used for auto-scaling provers and Witness Vector Generators.
+#[derive(Debug)]
+pub struct ProverQueueReporter {
+    config: FriProverGroupConfig,
+}
+
+impl ProverQueueReporter {
+    pub fn new(config: FriProverGroupConfig) -> Self {
+        Self { config }
+    }
+}
+
+#[async_trait]
+impl Task for ProverQueueReporter {
+    async fn invoke(&self, connection: &mut Connection<Prover>) -> anyhow::Result<()> {
+        let stats = connection
+            .fri_prover_jobs_dal()
+            .get_prover_jobs_stats()
+            .await;
+
+        for (protocol_semantic_version, circuit_prover_stats) in stats {
```

Comment:
Ah, I see what are you trying to do... But still, I don't see much value in still having this code in house keeper